src/server/license/check-answers/controller.js

```diff
@@ -0,0 +1,45 @@
+import { EmailAddress } from '../../common/model/answer/email-address.js'
+
+const indexView = 'license/check-answers/index.njk'
+export const pageTitle =
+  'Check your answers before you continue your application'
+export const heading = pageTitle
+
+/**
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const licenseCheckAnswersGetController = {
+  handler(req, h) {
+    const license = req.yar.get('license') ?? {}
+
+    const emailAddress = EmailAddress.fromState(
+      license?.emailAddress
+    ).validate()
```

Comment:
If we make the change as suggested above, this is a great place to use LicenceSection.fromState(req.yar.get('licence')

Reply:
Done